live/app/components/qroc-solution-panel.js

```diff
@@ -5,6 +5,21 @@ const QrocSolutionPanel = Ember.Component.extend({
   answer: null,
   solution: null,
 
+  isResultOk: Ember.computed('answer', function () {
+    const result = this.get('answer.result');
+    return result === 'ok';
```

Comment:
J'aurais fait : return this.get('answer.result') === 'ok';
Pour gagner une ligne vu que tu n'as qu'une seule instruction